R/article.R

```diff
@@ -238,6 +238,16 @@ ims_article <- function(journal = c("aoas", "aap", "aop", "aos", "sts"),
   )
 }
 
+#' @section `informs_article`: Format for creating submissions to
+#'   INFORMS journals. Adapted from \samp{https://pubsonline.informs.org/authorportal/latex-style-files}.
+#' @export
+#' @rdname article
+informs_article <- function(..., keep_tex = TRUE) {
+  pdf_document_format(
+    "informs", keep_tex = keep_tex, citation_package = "natbib", ...
+  )
+}
```

Comment:
As `natbib` seems to be enforced for this format, we could prevent future error if a user tries to set `citation_package`

Maybe 
```suggestion
informs_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
  citation_package <- match.arg(citation_package)
  pdf_document_format(
    "informs", keep_tex = keep_tex, citation_package = citation_package, ...
  )
}
```

or more verbose
```suggestion
informs_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
  if (citation_package != "natbib") {
    stop("Informs template only supports `natbib` for citation processing.")
  }
  pdf_document_format(
    "informs", keep_tex = keep_tex, citation_package = citation_package, ...
  )
}
```

What do you think ?

Reply:
Good idea.